packages/backend-common/config.d.ts

```diff
@@ -221,6 +224,16 @@ export interface Config {
       optionsSuccessStatus?: number;
     };
 
+    /**
+     * Content Security Policy options.
```

Comment:
sorry, unrelated change, just made order consistent with the backend-defaults file